pkg/provider/cloud/gcp/provider.go

```diff
@@ -336,6 +336,11 @@ func (g *gcp) ensureFirewallRules(cluster *kubermaticv1.Cluster, update provider
 		Build().
 		NodePorts()
 
+	allowedIPRange := cluster.Spec.Cloud.GCP.AllowedIPRange
+	if allowedIPRange == "" {
+		allowedIPRange = "0:0:0:0/0"
```

Comment:
Shouldn't this be `0.0.0.0/0`?

Reply:
absolutely, it's just a typo, going to fix it